AdyenDropIn/DropInComponent.swift

```diff
@@ -207,14 +210,13 @@ extension DropInComponent: PaymentComponentDelegate {
     /// :nodoc:
     public func didSubmit(_ data: PaymentComponentData, from component: PaymentComponent) {
         paymentInProgress = true
+        delegate?.didStart(component: component, from: self)
         delegate?.didSubmit(data, from: self)
     }
     
     /// :nodoc:
     public func didFail(with error: Error, from component: PaymentComponent) {
-        paymentInProgress = false
         if case ComponentError.cancelled = error {
-            stopLoading(withSuccess: false, completion: nil)
```

Comment:
StopLoading has to be called here for issuer list component and payment method list component to stop spinning if user cancels